lib/imports/convertCashnet.js

```diff
@@ -184,16 +184,24 @@ function parseGear(params, items, i) {
             refType.includes("ALL-S-")) {
             gearItem.size = refVal
         } else if (refType.includes("COLOR") ||
+                   refType.includes("MA4-SshirtUni") ||
+                   refType.includes("MA4-YTshirtC") ||
+                   refType.includes("MA4-WLSHIRT") ||
                    refType.includes("MA4-WBSHIRT") ||
+                   refType.includes("MA4-WBSHIRT") ||
```

Comment:
MA4-WBSHIRT is repeated, I trust these other gear codes are correct.

Reply:
good catch, ty. fixed